### ISSUE TITLE
terraformer-wkt-parser@1.1.1 breaking sequelize

### ISSUE THREAD

**Issue by trriplejay** [sequelize/sequelize#6455]
## What you are doing?

```js
npm install
```

## What do you expect to happen?
install dependencies that allow sequelize to function

## What is actually happening?
dependencies are installed, but one of them is causing an error.

`terraformer-wkt-parser` seems to have just released an update earlier today, and it seems to be breaking sequelize.  Previously, on terraformer-wkt-parser 1.1.0, everything was fine, and now that it has increated to 1.1.1, we see the below error.  This is confirmed by the PR opened by the greenkeeper bot here: https://github.com/sequelize/sequelize/pull/6454

_Output, either JSON or SQL_
```js
module.js:327
    throw err;
    ^
Error: Cannot find module 'terraformer-wkt-parser'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/travis/build/sequelize/sequelize/lib/data-types.js:7:13)
```

__Dialect:__  postgres 
__Sequelize version:__ 3.23.4

**Comment by felixfbecker** [comment]
Should be fixed by https://github.com/Esri/terraformer-wkt-parser/pull/24

**Comment by trriplejay** [comment]
confirmed that this is now fixed in terraformer-wkt-parser@1.1.2
